client/app/pages/queries/QuerySource.jsx

```diff
@@ -262,7 +262,7 @@ function QuerySource(props) {
                       data-executing={isQueryExecuting ? "true" : null}
                       syntax={dataSource ? dataSource.syntax : null}
                       value={query.query}
-                      schema={schema}
+                      schema={schema.length && schema[0].name ? schema : null}
```

Comment:
You can create a new branch fix this. The one branch just solve one problem. Thank you!